content/docs/actors/development/source_code.md

```diff
@@ -111,15 +116,15 @@ For example, for repositories on GitHub it can be done using the following steps
 https://api.apify.com/v2/acts/apify~hello-world/builds?token=<API_TOKEN>&version=0.1
 ```
 
-Then go to your GitHub repository, click **Settings**, select **Webhooks** tab and click **Add webhook**\. Paste the API URL to the **Payload URL** as follows:
+Then go to your GitHub repository, click **Settings**, select **Webhooks** tab and click **Add webhook**. Paste the API URL to the **Payload URL** as follows:
 
 ![GitHub integration]({{@asset actors/images/github-integration.webp}})
 
 And that's it! Now your actor should automatically rebuild on every push to the GitHub repository.
 
 ## [](#custom-environment-variables)Custom environment variables
 
-The actor owner can specify custom environment variables that are set to the actor's process during the run. Sensitive environment variables such as passwords or API tokens can be protected by setting the **Secret** option. With this option enabled, the value of the environment variable is encrypted and it will not be visible in the app or APIs, and the value is redacted from actor logs to avoid the accidental leakage of sensitive data.
+The actor owner can specify custom environment variables that are set to the actor's process during the run. Sensitive environment variables such as passwords or API tokens can be protected by setting the **Secret** option. With this option enabled, the value of the environment variable is encrypted, and it will not be visible in the app or APIs, and the value is redacted from actor logs to avoid the accidental leakage of sensitive data.
```

Comment:
```suggestion
The actor owner can specify custom environment variables that are set to the actor's process during the run. Sensitive environment variables such as passwords or API tokens can be protected by setting the **Secret** option. With this option enabled, the value of the environment variable is encrypted and it will not be visible in the app or APIs. In addition, the value is redacted from actor logs to avoid the accidental leakage of sensitive data.
```